docs/coordinates/frames.rst

```diff
@@ -42,11 +42,16 @@ is first created::
     >>> FK5()  # uses default equinox
     <FK5 Frame (equinox=J2000.000)>
 
-The specific names of attributes available for a particular frame (and
-their default values) are available as the class method
+The specific names of attributes available for a particular frame are available as the class method
 ``get_frame_attr_names``::
 
     >>> FK5.get_frame_attr_names()
+    dict_keys(['equinox'])
```

Comment:
Should you actually cast to a list or tuple to avoid having this be a dict_keys object?

Reply:
What is the disadvantage of that? It prints nicely, supports all the operations (most of them even faster then `tuple`) and doesn't make a copy necessary in performance critical code

Reply:
An argument can be made that the result here should be a `list` because of consistency with `Table.colnames`, which also returns a `list`. `dict_keys` could be useful in many cases, but they can be accessed directly as `frame_attributes.keys()`, similarly to what `Table.columns.keys()` provides.

Reply:
But then, for the mentioned performance benefits, all code currently using `get_frame_attr_names` should just use `frame_attributes.keys()` and there is actually no real reason to have `get_frame_attr_names` at all.

Reply:
All places where `get_frame_attr_names` is used internally in astropy are two cases:

a) iterating over the names
b) membership testing

both are faster when not making a list copy of the available dict keys

Reply:
If we do decide to replace all uses of `get_frame_attr_names()` with `frame_attributes.keys()` then that would at least provide a clean way of deprecating the former.

Reply:
done